utils/Recorder/strategies/TimelockStrategy.ts

```diff
@@ -0,0 +1,43 @@
+import { Strategy } from "./Strategy.interface";
+import { ContractReceipt, ContractTransaction, UnsignedTransaction } from "ethers";
+import { Timelock } from "../../../typechain";
+
+type QueuedTransaction = {
+  recipient: string;
+  value: number | string;
+  data: string;
+};
+
+//Strategy specific params
+export type TimelockParams = {
+  timelock: Timelock;
+  executionStartTimestamp: number;
+};
+
+// Implementation of Strategy.interface
+// Send transactions ony by one to the blockchain
+// need to specify during construction:
+// Signer - ethers.Signer connected to network
+export class TimelockStrategy implements Strategy {
+  params: TimelockParams;
+
+  constructor(params_: TimelockParams) {
+    this.params = params_;
+  }
+
+  async execute(txsToExecute: UnsignedTransaction[]): Promise<ContractReceipt[]> {
+    const toSend: QueuedTransaction[] = [];
+    while (txsToExecute.length > 0) {
```

Comment:
maybe we should check if `txsToExecute` is empty in the first place and return nothing or throw an error?